lib/release-message.js

```diff
@@ -19,35 +20,27 @@ module.exports = function buildReleaseMessage () {
    * @param {Array} labels
    */
   function addLine (message, labels = []) {
-    if (labels.includes('bugfix')) {
-      semverBuckets.bugfix.push(message)
-    } else if (labels.includes('semver-minor')) {
-      semverBuckets.feature.push(message)
-    } else if (labels.includes('semver-major')) {
-      semverBuckets.breakingChanges.push(message)
-    } else {
-      semverBuckets.mixed.push(message)
-    }
+    semverBuckets.list.push(message)
+
+    // if (labels.includes('bugfix')) {
+    //   semverBuckets.bugfix.push(message)
+    // } else if (labels.includes('semver-minor')) {
+    //   semverBuckets.feature.push(message)
+    // } else if (labels.includes('semver-major')) {
+    //   semverBuckets.breakingChanges.push(message)
+    // } else {
+    //   semverBuckets.mixed.push(message)
+    // }
   }
 
   function toString () {
-    return printChangelog`🌟 Breaking changes:
-${semverBuckets.breakingChanges}
-
-🚀 Features:
-${semverBuckets.feature}
-
-🐛 Fixed:
-${semverBuckets.bugfix}
-
-🔍 Others:
-${semverBuckets.mixed}`
+    return printChangelog`📚 PR: ${semverBuckets.list}`
```

Comment:
I think we should not use a tagged template here, mainly because I would like to load the template.